changelogs/unreleased/4630-izturn-minor.md

```diff
@@ -0,0 +1,3 @@
+## Unify the ensure objects deleted
```

Comment:
Let's switch this to a `small` changelog (file name needs to be updated), and drop the header here, just include the line of text below.

Reply:
done